README.md

```diff
@@ -425,21 +323,36 @@ You can lint your code independently of this by running:
 $ npm run lint
 ```
 
+## Logging
+
+This project uses Winston for serverside logging, and New Relic for clientside logging. Serverside logs for this project's `qa` and `production` branches can be found in AWS Cloudwatch under the nypl-dams-dev and nypl-dams-prod accounts, and clientside logs can be found in New Relic (Digital Collections (prod/qa) > Browser > errors). For Vercel deployments, logging will appear in the console.
+
+We structure our logs according to these [NYPL standards](https://github.com/NYPL/engineering-general/blob/main/standards/logging.md). See `logger.js` for the formatting and storage of logs.
+
+Do NOT use `console.log` if you want to add a permanent log, and make sure to clean up your debugging `console.log`s, because they will clutter Cloudwatch and New Relic.
```

Comment:
I like the language used, "NOT", "DO". Don't change it, but I wonder if there's a way to use what's in engineering-general or perhaps we should extend engineering-general when it says

> The key words MUST, MUST NOT, SHOULD, SHOULD NOT, and MAY/OPTIONAL, in these documents are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).